example/search.dart

```diff
@@ -16,7 +16,7 @@ void main() async {
   await page.click(allResultsSelector);
 
   // Wait for the results page to load and display the results.
-  const resultsSelector = '.gsc-results .gsc-thumbnail-inside a.gs-title';
+  const resultsSelector = '.gsc-result a.gs-title';
```

Comment:
This test sometimes fails also on Chromium.
It seems that the google result page sometimes returns the page with different DOM structure.